appveyor.yml

```diff
@@ -7,23 +7,34 @@ environment:
       PYTHON: "C:\\conda"
       MINICONDA_VERSION: "latest"
       CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
-      ASTROPY_VERSION: "development"
+      NUMPY_VERSION: "stable"
       CONDA_DEPENDENCIES: "semantic_version jsonschema pyyaml six lz4"
       PIP_DEPENDENCIES: "git+http://github.com/spacetelescope/gwcs.git#egg=gwcs"
 
   matrix:
       - PYTHON_VERSION: "2.7"
-        NUMPY_VERSION: "1.9.1"
+        ASTROPY_VERSION: "stable"
         platform: x86
         PYTHON_ARCH: "32"
 
       - PYTHON_VERSION: "2.7"
-        NUMPY_VERSION: "1.9.1"
+        ASTROPY_VERSION: "stable"
         platform: x64
         PYTHON_ARCH: "64"
 
       - PYTHON_VERSION: "3.4"
-        NUMPY_VERSION: "1.9.1"
+        ASTROPY_VERSION: "stable"
+        platform: x64
+        PYTHON_ARCH: "64"
+
+      - PYTHON_VERSION: "3.5"
+        ASTROPY_VERSION: "1.3.2" "stable" "development"
```

Comment:
Is this valid syntax?

Reply:
I'm actually not sure. I thought I saw an example along these lines somewhere but this commit was kind of just an experiment.

Reply:
I assume it should be valid yaml but there may be other things. I saw there's a validation tool here https://ci.appveyor.com/tools/validate-yaml

Reply:
The [appveyor validator](https://ci.appveyor.com/tools/validate-yaml) says no, so I guess it's back to the drawing board. I was hoping to avoid listing each configuration explicitly, but I'm not sure that will be possible.